Matlab/Matlab.sublime-syntax

```diff
@@ -467,10 +467,11 @@ contexts:
     - include: block-comments
 
   line-comments:
-    - match: (%%(?!%)).*\n?
+    - match: ^\s*(%%(?!%))\s*(.*\S)?\s*\n?
```

Comment:
is `^` necessary?

Reply:
According to the link above
> You also can enter two percent signs (%%) at the start of the line where you want to begin the new section.

(but they also work after whitespace nested within e.g. loops to create section breaks)